lib/pure/osproc.nim

```diff
@@ -410,6 +410,45 @@ when defined(Windows) and not defined(useNimRtl):
       copyMem(addr(result[L]), cstring(x), x.len+1) # copy \0
       inc(L, x.len+1)
 
+  proc envToWideCString(env: StringTableRef): WideCString =
+    # newWideCString stops on \0 characters, so we have to
```

Comment:
True. But it still stops on \0 char (see `runes` iterator). Didn't want to break any code depending on it.